docs/docs/introduction/getting-started.mdx

```diff
@@ -64,54 +97,64 @@ To start all the default Instant OpenHIE packages, select a deployment platform
     ```
 
 1. To stop the running containers, execute the following:
 
     ```sh
-    ./instant.sh down docker
+    yarn docker:instant down
     ```
 
 1. Remove the containers with the command below. However, make sure you **stop all the containers before trying to delete** them.
 
     > **This action will also delete all volumes** created by the containers.
 
     ```sh
-    ./instant.sh destroy docker
+    yarn docker:instant destroy
     ```
 
+Each command also takes a list of package IDs to operate on. If this is left out then all packages are run by default.
+
+If you do not have the code checked-out then you can still startup the environment with the correct command. You can find the docker commands to run in the scripts section of [this file](https://github.com/openhie/instant/blob/ohie-420-docker-outside-of-docker/package.json) in the codebase. Add the action you would like to perform (init, up, down, destroy) to the end of that command.
+
 </TabItem>
 <TabItem value="kubernetes">
 
-**Before proceeding**, make sure you are in the main `/instant` directory. This directory contains the `instant.sh` script.
+For this to work you will need to have Kubernetes enabled in Docker for Desktop or minikube installed and running for Linux.
 
-1. To start up the system, execute the following command:
+If you have the Instant [OpenHIE codebase](https://github.com/openhie/instant) checked-out make sure you are in the main `/instant` directory.
 
-    > This command will output urls from which you can access your Instant OpenHIE instance.
+1. To start up the system, execute the following command:
 
     ```sh
-    ./instant.sh up k8s
+    yarn
+    yarn docker:instant init -t kubernetes
     ```
 
-1. To delete all the deployment related pods, run the command below.
-
-    > This command will leave services, and volumes intact.
+1. To stop the running containers, execute the following:
 
     ```sh
-    ./instant.sh down k8s
+    yarn docker:instant down -t kubernetes
     ```
 
-1. To purge the entire Instant OpenHIE system, run the command below.
+1. Remove the containers with the command below. However, make sure you **stop all the containers before trying to delete** them.
+
+    > **This action will also delete all volumes** created by the containers.
 
     ```sh
-    ./instant.sh destroy k8s
+    yarn docker:instant destroy -t kubernetes
     ```
 
+Each command also takes a list of package IDs to operate on. If this is left out then all packages are run by default.
+
+If you do not have the code checked-out then you can still startup the environment with the correct command. You can find the docker commands to run in the scripts section of [this file](https://github.com/openhie/instant/blob/ohie-420-docker-outside-of-docker/package.json) in the codebase. Add the action you would like to perform (init, up, down, destroy) to the end of that command and make sure to add the `-t kubernetes` option after the action.
```

Comment:
Maybe we can update this link so long to point to the location of where it will be once in master, to avoid having another PR for correcting the file location